src/lib/holocene/copyable/button.svelte

```diff
@@ -29,6 +29,6 @@
 
 <style lang="postcss">
   .copy-button {
-    @apply m-1 rounded-md border-2 border-[transparent] hover:border-indigo-600 hover:bg-gradient-to-br hover:from-blue-100 hover:to-purple-100 hover:shadow-focus hover:shadow-blue-600/50 focus-visible:border-indigo-600 focus-visible:bg-gradient-to-br focus-visible:from-blue-100 focus-visible:to-purple-100 focus-visible:shadow-focus focus-visible:shadow-blue-600/50 focus-visible:outline-none;
+    @apply m-1 rounded-md border-2 border-[transparent] bg-transparent text-primary hover:surface-interactive-secondary focus-visible:surface-interactive-secondary focus-visible:border-indigo-600 focus-visible:outline-none;
```

Comment:
Looks like `text-primary` is causing the icon to "disappear" in light mode

```suggestion
    @apply m-1 rounded-md border-2 border-[transparent] bg-transparent hover:surface-interactive-secondary focus-visible:surface-interactive-secondary focus-visible:border-indigo-600 focus-visible:outline-none;
```

| Before | After |
|--|--|
|![Screenshot 2024-04-11 at 11 49 18 AM](https://github.com/temporalio/ui/assets/15069288/6abfbe30-c0ad-48b2-ab1e-11f4aae93b28)|![Screenshot 2024-04-11 at 11 51 25 AM](https://github.com/temporalio/ui/assets/15069288/7ded79be-f250-4a6c-a39c-3cf56b73e83a)|